### ISSUE TITLE
Layout issues on tools bar at the top

### ISSUE THREAD

**Issue by linkfang** [openhwgroup/openhwgroup.org#424]
Issues found by @DBees 

> While we're at it can you look at the create account section (which is actually aimed at creating an EF account). First question is why is duplicated in the green bar? Shouldn't that be removed?

> "Create Account" and Edit Account" lead to exactly the same material. Why not replace it with a single "Create or Edit Eclipse Foundation Account"

> Third question why don't we put that under "resources" instead of a separate menu bar

> Fourth question "Manage cookies" doesn't do anything, why don't we get rid of it, unless you are planning to introduce a cookie management function.

**Comment by autumnfound** [comment]
> While we're at it can you look at the create account section (which is actually aimed at creating an EF account). First question is why is duplicated in the green bar? Shouldn't that be removed?

> "Create Account" and Edit Account" lead to exactly the same material. Why not replace it with a single "Create or Edit Eclipse Foundation Account"

This is an issue with this site as we reimplement the toolbar, as the first instance of the toolbar properly routes users to the right pages. We should be just setting the toolbar background to white rather than reimplementing it.

> Third question why don't we put that under "resources" instead of a separate menu bar

We can definitely add another callout to this page under resources if that's the wish of the team! We just update with whatever information is passed to us, and the create account page was always left in the nav.

> Fourth question "Manage cookies" doesn't do anything, why don't we get rid of it, unless you are planning to introduce a cookie management function.

It does do something, if you've accepted cookies already ;) Current production behaviour is that it triggers the cookie banner at the bottom of the page and unsets the cookie acceptance. This is in line with the rest of our sites and is inline with GDPR.

**Comment by DBees** [comment]
Unless there is a reason not to, I recommend we put the "create account" under the resources menu, and call it "Create Eclipse Foundation Account". As is it implies there is an OpenHW Account function which is inaccurate and confusing.  

> This is an issue with this site as we reimplement the toolbar, as the first instance of the toolbar properly routes users to the right pages

So are you saying there is some different function called Edit Account?


> It does do something, if you've accepted cookies already ;) Current production behaviour is that it triggers the cookie banner at the bottom of the page and unsets the cookie acceptance. This is in line with the rest of our sites and is inline with GDPR

Why does it unset cookie acceptance? I don't understand what is going on here with cookies. What happens if you select "Allow cookies"? 


I tried the manage cookies again and "allowed cookies" at the bottom of page.

**Comment by autumnfound** [comment]
> Unless there is a reason not to, I recommend we put the "create account" under the resources menu, and call it "Create Eclipse Foundation Account". As is it implies there is an OpenHW Account function which is inaccurate and confusing.

@chrisguindon I don't think this impacts us to remove that functionality for now right?

> So are you saying there is some different function called Edit Account?

Yep! If you are logged in, you can click edit account and it will bring you to a profile edit for the logged in user on accounts.eclipse.org. That's our accounts management site, and where people log in as well.

> Why does it unset cookie acceptance? I don't understand what is going on here with cookies. What happens if you select "Allow cookies"?

This is a GDPR thing as people have the right to "opt-out" of cookies. This is because there are tracking cookies we use to improve experience, or to enable things like Hubspot or some Google functionalities. If you opt-out, the idea is that we don't allow the code that enables those features to run. If we didn't have that it could lead to legal problems. 

If you want more info on cookie use laws, and some of the exceptions, see https://siteimprove.com/en-ca/gdpr/gdpr-cookie-consent-policy/. Its a bit technical, but it gives a really good overview of why we do it the way we do.

**Comment by chrisguindon** [comment]
@autumnfound I will try to respond by responding to each issue reported in the initial comment.

> While we're at it can you look at the create account section (which is actually aimed at creating an EF account). First question is why is duplicated in the green bar? Shouldn't that be removed?

+1 That's a bug. It shouldn't be duplicated. Let's remove the first one in the green bar.

> "Create Account" and Edit Account" lead to exactly the same material. Why not replace it with a single "Create or Edit Eclipse Foundation Account"

I am fine with this change.

> Third question why don't we put that under "resources" instead of a separate menu bar

I recommend that we keep it at the top of our page. We try to be consistent across our web properties with these links.

> Fourth question "Manage cookies" doesn't do anything, why don't we get rid of it, unless you are planning to introduce a cookie management function.

@autumnfound got this right!

The manage cookie is very important and must stay. It allows users to change their decision using our cookie consent banner which is a GDPR requirement.

**Comment by DBees** [comment]
> Third question why don't we put that under "resources" instead of a separate menu bar
>> I recommend that we keep it at the top of our page. We try to be consistent across our web properties with these links
I don't like the barnacle aspect of this. The website should provide  access to resources in a consistent way, not with small and easily forgotten special purpose menus

**Comment by chrisguindon** [comment]
Is there question there @DBees? I recommend that we keep at the top for consitency. That's what we do for all our sites, so users are used to seeing that link.

However, it's not a requirement. If you really want us to make that change, we will do so.

**Comment by DBees** [comment]
@chrisguindon seems the consistency issue would be mainly for the website maintainers, not for the OpenHW members. However, I sense you don't think this is a wise change so I'll defer to your judgement. Let's fix the bug as noted, consolidate the buttons into "Create or Edit" as noted above, and leave Manage Cookies in place. Thanks.

**Comment by chrisguindon** [comment]
> @chrisguindon seems the consistency issue would be mainly for the website maintainers, not for the OpenHW members.

Not really. OpenHWGroups community members do need to interact with some of our Eclipse Foundation websites for various things such as signing the ECA.

It's really not difficult for us to make that change. If you are asking my opinion, I would keep it at the top but if you prefer we move it, we can definitely do that.

**Comment by DBees** [comment]
K great so I would say put in the resources menu please. Thanks :)

**Comment by linkfang** [comment]
Hi @DBees , please preview the updated version here:
https://deploy-preview-435--eclipsefdn-openhwgroup.netlify.app/

Things being updated:

- Removed the duplicated toolbar at the top (in green background)
- Combined "Create account" and "Edit account" into "Create or Edit Account" (I removed "Eclipse Foundation" as it will make the menu item too long
- Moved the combined menu item from toolbar to Resources (1st item).

**Comment by DBees** [comment]
Looks good @linkfang. Since Eclipse Foundation is too long, how about "Create or Edit Eclipse Account".

**Comment by linkfang** [comment]
It fits in perfectly!

https://deploy-preview-435--eclipsefdn-openhwgroup.netlify.app/

**Comment by DBees** [comment]
Yes, Also can you make the "Blog" entry similar to the others ie "OpenHW Blog"

**Comment by linkfang** [comment]
Sure, updated it:
https://deploy-preview-435--eclipsefdn-openhwgroup.netlify.app/

**Comment by DBees** [comment]
Looks good, thank you @linkfang !

**Comment by linkfang** [comment]
NP! The changes are now live!
